UpskillWikipediaUITests/TestSuites/UpskillTests.swift

```diff
@@ -0,0 +1,57 @@
+import Foundation
+import XCTest
+
+class UpskillTests: BaseTestCase {
+    
+    lazy var homePage = HomePage(application: self.app)
+    lazy var searchPage = SearchPage(application: self.app)
+    lazy var articlePage = ArticlePage(application: self.app)
+    lazy var settingsPage = SettingsPage(application: self.app)
+    lazy var webViewPage = WebViewPage(application: self.app)
+    
+    func testSearchForTermAndDisplayArticle_TC1() {
+        let searchQuery = "Tomato"
+        
+        homePage.tapSearchFiled()
+        searchPage.typeIn(searchQuery: searchQuery)
+        searchPage.tapFirstResult()
+        
+        XCTAssertTrue(articlePage.isTextOnPageDispplayed(text: searchQuery), "Article page for search quesry didn't open")
```

Comment:
Typo in assertion message. Also it's not a bad idea to add the value of the search query to the message is with assertion for boolean values you won't know what type of query the test was using until you look into the code.
```
XCTAssertTrue(articlePage.isTextOnPageDispplayed(text: searchQuery), "Article page for query '\(searchQuery)' didn't open")
```